contracts/src/periphery/EigenDACertVerifierV2.sol

```diff
@@ -32,9 +32,6 @@ contract EigenDACertVerifierV2 is IEigenDACertVerifierV2 {
     /// @notice The EigenDASignatureVerifier contract address
     IEigenDASignatureVerifier public immutable eigenDASignatureVerifierV2;
 
-    /// @notice The EigenDA middleware OperatorStateRetriever contract address
-    OperatorStateRetriever public immutable operatorStateRetrieverV2;
```

Comment:
I guess the original thinking with this contract is that we'd have a single one shared by all AVSs so we'd save on deployment gas costs by not having every single contract deploy it as an embedded library.

But if its immutable pointer then agree that it doesn't make a lot of sense, unless we're reaching the bytesize limit of the contract. So I think I'm fine with this change although I don't know the solidity best practices.

Reply:
I think it is important to reduce dependencies on "deployed" contracts, as this is a prerequisite to lessening the number of contracts we have to maintain overall.